plugin/storage/es/options.go

```diff
@@ -233,7 +236,7 @@ func initFromViper(cfg *namespaceConfig, v *viper.Viper) {
 	cfg.Password = v.GetString(cfg.namespace + suffixPassword)
 	cfg.TokenFilePath = v.GetString(cfg.namespace + suffixTokenPath)
 	cfg.Sniffer = v.GetBool(cfg.namespace + suffixSniffer)
-	cfg.servers = stripWhiteSpace(v.GetString(cfg.namespace + suffixServerURLs))
+	cfg.Servers = strings.Split(stripWhiteSpace(v.GetString(cfg.namespace+suffixServerURLs)), ",")
```

Comment:
IIRC there's an easier way to do this with viper, using GetStringSlice.

Reply:
I could not make it work with the format of expected values `url`, url2`